src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -190,28 +190,28 @@
     <value>Mark assemblies with assembly version</value>
   </data>
   <data name="MarkAssembliesWithComVisibleTitle" xml:space="preserve">
-    <value>Mark assemblies with ComVisible</value>
+    <value>Mark assemblies with 'ComVisible'</value>
```

Comment:
@rainersigwald We are considering updating all analyzer titles and messages to treat API terms the same way we treat placeholders for type or member names, with single quotes around them. As a person particularly interested in English grammar, I would value your opinion on this proposal.

Reply:
I think it makes sense and is consistent. :shipit: